website/sidebars.js

```diff
@@ -10,6 +10,38 @@
  */
 
 module.exports = {
+  '0.3-go-sdk': [
+    {
+      type: "category",
+      label: "Dagger SDK for Go",
```

Comment:
```suggestion
      label: "Go SDK",
```

Short form, per discussion